pallas-network/src/miniprotocols/localstate/queries_v16/mod.rs

```diff
@@ -80,6 +84,30 @@ pub struct SystemStart {
     pub picoseconds_of_day: u64,
 }
 
+#[derive(Debug, Encode, Decode, PartialEq)]
+pub struct StakeDistribution {
+    #[n(0)]
+    pub pools: KeyValuePairs<Bytes, Pool>,
+}
+
+#[derive(Serialize, Deserialize, Debug, Encode, Decode, PartialEq, Clone)]
```

Comment:
why do we need Serialize / Deserialize on these structs?

Reply:
nice catch, this is debug dirty stuff. removing